e2e/interchaintestv8/types/solidity_fixtures.go

```diff
@@ -13,16 +13,35 @@ import (
 	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/testvalues"
 )
 
+// Global variable to hold the genesis fixture
+var sp1GenesisFixture *Sp1GenesisFixture
```

Comment:
Don't love this. Gloabl variables are bad :/
Can't the tests just hold this?

Reply:
fixed